src/app/loadout/loadout-edit/LoadoutEditBucket.tsx

```diff
@@ -117,6 +117,7 @@ export function ArmorExtras({
 }) {
   const equippedItems =
     items?.filter((li) => li.loadoutItem.equip && !li.missing).map((li) => li.item) ?? [];
+  const anyMissing = Boolean(items?.some((li) => li.loadoutItem.equip && li.missing));
```

Comment:
So it only says "Choose Armor" if the loadout needs replacement items -- I think "Choose Armor" would be better than "Optimize Armor" also if the loadout just doesn't specify an equipped item for an armor bucket, i.e. maybe the condition should be "fewer than 5 resolved equipped armor items"?

Reply:
Good point - I was envisioning the "shared loadout" case but it could also apply to the underspecified loadout.